tests/test_core.py

```diff
@@ -1652,19 +1652,19 @@ class myexception : public exception {
                   // the exception, if necessary. By incrementing and decrementing the refcount
                   // we trigger the free'ing of the exception if its refcount was zero.
                   ___cxa_increment_exception_refcount(p);
-                  console.log(Module["formatException"](p).replace(/0x[0-9a-f]*/, "xxx"));
```

Comment:
@hoodmane By the way I have a question. Is there a reason we can't call this just like
```js
formatException(p);
```
?

The same for `Module["_throw_exc"]` in line 1648.

Reply:
I'm not sure, try it and see?

Reply:
From outside the module you would need `Module["formatException"]`.. but for JS that is *inside* the module (e.g. EM_ASM and EM_JS code) then you can just do `formatException`.

Reply:
Thanks. Changed them to direct calls.